### ISSUE TITLE
(Novice Issue)  /capt0000.arw  on Sony A6500

### ISSUE THREAD

**Issue by donnysnarko** [gphoto/gphoto2#404]
Hello,
Ive been trying to configure my A6500 to work with gphoto on Octoprint via my Raspberry Pi.  Ive been following the instructions [here](https://github.com/FormerLurker/Octolapse/wiki/Configuring-an-External-Camera), but have hit a problem where the camera clearly takes a photo when prompted with **gphoto2 --capture-image** but then indicates it has been stored on the camera at **/capt0000.arw** .  Ime very new to using a Pi (i'm not a programmer at all), but I cannot find the **/capt0000.arw** file on the camera (i cannot essentially find teh saved image) and i suspect that when i capture multiaple image it just overwites teh existing one.  Can somsone lend a noob a hand here and point me to something that may help?

**Comment by msmeissn** [comment]
This filename is a virtual filename, it only lives in camera and then in gphoto2 RAM.

You need to "download it" right after capture.

If you want to leave it on the SD Card, you can switch "capturetarget" on the A6500 I think.

**Comment by donnysnarko** [comment]
You are correct, i need to switch it to capturetarget, currenly it is set to capture like this
-
Sony Alpha-A6500 (control)    usb: 001, 016
Label: Capture
Readonly: 0
Type: TOGGLE
Current: 2
END
-
Do you know how i am able to switch to capture target? 
on [the guide im following](https://github.com/FormerLurker/Octolapse/wiki/V0.4---Configuring-a-DSLR-With-No-Download) it suggest that I change this to capturetargetwithin the Raspberry PI  - heres the relevant section


**Determine the appropriate capturetarget setting
The config options vary by camera and by the gphoto2 version. You can run the following command to see what options for capturetarget are available on your camera:

gphoto2 --auto-detect --get-config capturetarget
Here's what my camera reports after running this command:

_Label: Capture Target_
Readonly: 0
Type: RADIO
Current: Memory card
Choice: 0 Internal RAM
Choice: 1 Memory card
END
If your options aren't the same as mine (Memory card = 1) you will have to adjust the initialize-camera-save-to-sd.sh script accordingly.**

**Comment by msmeissn** [comment]
can you post or attach output of

gphoto2 --summary

gphoto2 --list-all-config

**Comment by donnysnarko** [comment]
> can you post or attach output of
> 
> gphoto2 --summary
> 
> gphoto2 --list-all-config

Thanks.  
Here's the [Summary](https://drive.google.com/file/d/111vLlwHvRDsB-EhLAB79JxGCtacA4Ce8/view?usp=sharing)

and here's the [List-all-config](https://drive.google.com/file/d/11-jCyoNMqds8L0Q7KI0Y53g-RX8xULHN/view?usp=sharing)

**Comment by donnysnarko** [comment]
Hi @msmeissn, i hope the info i provided above is enough to go on

**Comment by msmeissn** [comment]
can you try 
gphoto2 --set-config d222=17 --capture-image

and see if it stays in the camea card?

**Comment by msmeissn** [comment]
(This is decoded as "capturetarget" option in libgphoto2 2.5.24 or later, but octoprint might be a bit behind in versions.)

**Comment by donnysnarko** [comment]
> (This is decoded as "capturetarget" option in libgphoto2 2.5.24 or later, but octoprint might be a bit behind in versions.)

Hello, ive tried that adn it didnt work.  Here's a video to show the steps
https://drive.google.com/file/d/11AKz8b6e30PhuccS1Lb9x439pIVAW1hJ/view?usp=sharing

**Comment by donnysnarko** [comment]
hi again @msmeissn  , from what yove seen in my video do you think it just wont work with my camera, or are there some other checks I can do?  Thank you for your help so far

**Comment by msmeissn** [comment]
I currently have not much further ideas... Your video looks correct :/

**Comment by donnysnarko** [comment]
> I currently have not much further ideas... Your video looks correct :/

It's odd, but it's just not saving to my camera sd card.
